expyfun/tests/test_experiment_controller.py

```diff
@@ -439,6 +439,15 @@ def test_button_presses_and_window_size():
         fake_button_press(ec, '1', 0.3)
         out = ec.wait_for_presses(1.5, live_keys=['1'], timestamp=False)
         assert_equal(out[0], '1')
+        fake_button_press(ec, 'a', 0.3)
+        fake_button_press(ec, 'return', 0.5)
+        assert_equal(ec.text_input(), 'A')
```

Comment:
FYI we no longer need to do `assert_equal`, with `pytest` you can do `assert a == b` directly and it does good things automagically